main.cpp

```diff
@@ -118,11 +114,7 @@ double DoubleType::divide(double lhs, double rhs)
         std::cout << "\nwarning, floating point division by zero returns 'inf' !\n";
         return lhs / rhs;
     }
-
-    else
-    {
-        return lhs / rhs;
-    }
+    return lhs / rhs;
```

Comment:
You are still repeating yourself here.  D.R.Y.  
Don't Repeat Yourself
- [ ] revise